srml/democracy/src/lib.rs

```diff
@@ -86,13 +88,13 @@ decl_module! {
 		/// Propose a sensitive action to be taken.
 		fn second(origin, proposal: Compact<PropIndex>) -> Result {
 			let who = ensure_signed(origin)?;
-			let proposal: PropIndex = proposal.into();
-			let mut deposit = Self::deposit_of(proposal)
+			let index: PropIndex = proposal.into();
```

Comment:
bad rename. instance identifiers should generally identify by *instance* not by *class*.